mindmeld/components/entity_resolver.py

```diff
@@ -264,7 +486,7 @@ def fit(self, clean=False):
                     "without ID."
                 )
             logger.info("Importing synonym data to knowledge base index '%s'", kb_index)
-            EntityResolver.ingest_synonym(
+            ElasticsearchEntityResolver.ingest_synonym(
```

Comment:
do we still need this to be a class method?

Reply:
I think not required. Will make that change.

Reply:
should it be `self.ingest_synonym()` to be consistent with L403?